api/urls.py

```diff
@@ -7,5 +7,6 @@
 urlpatterns=[
 	url(r'^projects/$',views.projects,name='projects'),
 	url(r'^projects/(?P<project_slug>[\w-]+)/$',views.project_details,name='project_detail'),
-	url(r'^members/$',views.members,name="members")
+	url(r'^members/$',views.members,name="members"),
+	url(r'^test$',views.test),
```

Comment:
I made them for the local check, I have removed them in the last commit
@mbad0la  thanks for pointing out :+1: